lib/tools/adb-commands.js

```diff
@@ -1021,12 +1022,13 @@ methods.removePortForward = async function (systemPort) {
  */
 methods.getReverseList = async function () {
   log.debug(`List reverse forwarding ports`);
-  let connections = await this.adbExec(['reverse', '--list']);
-  return connections.split('\n');
+  const connections = await this.adbExec(['reverse', '--list']);
+  return connections.split(EOL);
```

Comment:
shall we make sure there are no empty lines here if trimmed?

`return connections.split(EOL).filter((line) => !!line.trim())`

Reply:
same comment to the other method